docs/sop/PROVISIONING_PRODUCTION.md

```diff
@@ -0,0 +1,490 @@
+
+
+Next, Initialise Ansible including copying ansible files into `staging` environment and save to Gitlab variables the IP addresses output by terraform with this command:
+```  
+../../../scripts/ansible_init.sh --env staging  
+```  
+
+Install third party Ansible roles:
+```  
+ansible-galaxy install -r ../../../infrastructure/requirements.yml  
+```  
+
+Provision the bastion server:
+```  
+env OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES ansible-playbook -i ../../inventories bastion_playbook.yml --extra-vars="gigadb_env=staging"  
+```  
+
+Provision web server:
+```  
+env TF_KEY_NAME=private_ip OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES ansible-playbook -i ../../inventories webapp_playbook.yml --extra-vars="gigadb_env=staging"
+```  
+
+Provision files server:
+```  
+env TF_KEY_NAME=private_ip OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES ansible-playbook -i ../../inventories files_playbook.yml --extra-vars="gigadb_env=staging"  
+```
+
+
+At this stage you can trigger a pipeline for the `develop` branch on the `upstream/gigadb-website` Gitlab project:
+
+Go to [Gitlab Upstream pipeline page](https://gitlab.com/gigascience/upstream/gigadb-website/-/pipelines)
+and run all the jobs in the staging build stage in your pipeline, and press "Run pipeline",
+then select`develop` and confirm   by clicking "Run pipeline".
+This will execute all automated jobs.
+There is a couple of manual jobs that will also need triggering: `PureFtpdBuildStaging` and then `PureFtpdDeployStaging`.
+
+
+When the manual and automated jobs have all completed successfully, it will result in a partial deployment of GigaDB website to the infrastructure we've just provisioned.
+Finally, you can perform the last step which is to load the environment's database server with data and install the tools on bastion servers needed by the users:
+
+```
+env OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES ansible-playbook -i ../../inventories data_cliapp_playbook.yml -e "gigadb_env=staging"
+```
+
+The website should be visitable at https://staging.gigadb.org, and the bastion server ready at bastion-stg.gigadb.host.
+
+## Staging for upstream/alt-gigab-website
+
+The provisioning for this project is similar to the other one, except that you need changing the key parameters that are
+described in the table in the "Upstream projects" section at the top of `docs/sop/DEPLOYING_TO_PRODUCTION.md`
+
+Change directory to the local checkout of the `upstream/alt-gigadb-website` project:
+
+```
+cd gigadb-alt-upstream
+```
+
+Change directory to the `envs` folder:
+```  
+cd ops/infrastructure/envs  
+```  
+
+Create directory `staging` directory if not existing already and change to it:
+```  
+mkdir staging
+```
+```  
+cd staging  
+```  
+
+Initialise Terraform, including creating a new state (or retrieving if existing) on Gitlab and copying terraform files to `staging` environment:
+```   
+../../../scripts/tf_init.sh --project gigascience/upstream/gigadb-website --env staging  
```

Comment:
Hi @rija,

**typo:** The project should be `gigascience/upstream/alt-gigadb-website`